### ISSUE TITLE
unrecognized arguments: twitch.tv/name best

### ISSUE THREAD

**Issue by GoToLoop** [streamlink/streamlink-twitch-gui#871]
### Checklist

- [X] I understand that **Streamlink Twitch GUI** is just a launcher for **Streamlink**
- [X] [I have read the contribution guidelines](https://github.com/streamlink/streamlink-twitch-gui/blob/master/CONTRIBUTING.md)
- [X] [I have checked the list of open and recently closed issues](https://github.com/streamlink/streamlink-twitch-gui/issues?q=is%3Aissue+sort%3Aupdated-desc)
- [X] [I have checked the commit log of the master branch](https://github.com/streamlink/streamlink-twitch-gui/commits/master)

### Streamlink Twitch GUI version

v2.0.0

### Streamlink version

v3.1.1

### Operating system, environment and configuration details

Windows 8.1

### Description

After upgrading today for the latest v2.0.0 version, b/c the old 1 stopped working due to API change, I'm having this message "streamlink: error: unrecognized arguments: twitch.tv/name best" every time I attempt to launch a stream.

### Debug log

```text
[2022-02-16T23:33:45.518Z][debug][Application]
Parameters
{
    "_": [],
    "tray": false,
    "hide": false,
    "hidden": false,
    "max": false,
    "maximize": false,
    "maximized": false,
    "min": false,
    "minimize": false,
    "minimized": false,
    "reset-window": false,
    "versioncheck": true,
    "version-check": true,
    "logfile": true,
    "loglevel": "debug",
    "l": "debug",
    "launch": "",
    "goto": ""
}

[2022-02-16T23:34:09.508Z][debug][StreamingService]
Preparing to launch stream
{
    "id": "30666385",
    "quality": "source",
    "low_latency": false,
    "disable_ads": true,
    "chat_open": false,
    "started": "2022-02-16T23:34:09.486Z",
    "stream": "30666385",
    "user": "30666385"
}

[2022-02-16T23:34:09.516Z][debug][StreamingService]
Resolving streaming provider
{
    "provider": "streamlink",
    "providerUserData": {
        "exec": "",
        "params": "",
        "pythonscript": null
    }
}

[2022-02-16T23:34:09.522Z][debug][StreamingService]
Found streaming provider
{
    "exec": "C:\\Program Files (x86)\\Streamlink\\bin\\streamlinkw.exe",
    "params": null,
    "env": null
}

[2022-02-16T23:34:09.535Z][debug][StreamingService]
Spawning process
{
    "exec": "C:\\Program Files (x86)\\Streamlink\\bin\\streamlinkw.exe",
    "params": [
        "--version"
    ],
    "env": null
}

[2022-02-16T23:34:09.937Z][debug][StreamingService]
Validated streaming provider
{
    "version": "3.1.1"
}

[2022-02-16T23:34:09.973Z][debug][StreamingService]
Spawning process
{
    "exec": "C:\\Program Files (x86)\\Streamlink\\bin\\streamlinkw.exe",
    "params": [
        "--twitch-disable-hosting",
        "--twitch-disable-ads",
        "--player-no-close",
        "--hls-live-edge",
        "2",
        "--stream-segment-threads",
        "4",
        "--retry-open",
        "1",
        "--retry-streams",
        "1",
        "twitch.tv/streamer_name",
        "best"
    ],
    "env": null
}

[2022-02-16T23:34:10.639Z][error][StreamingService]
Error: usage: streamlink [OPTIONS] <URL> [STREAM]
{
    "id": "30666385",
    "quality": "source",
    "low_latency": false,
    "disable_ads": true,
    "chat_open": false,
    "started": "2022-02-16T23:34:09.486Z",
    "stream": "30666385",
    "user": "30666385"
}
```

**Comment by bastimeyer** [comment]
This is not a Streamlink Twitch GUI error message, this is a Streamlink error message. What you have posted here doesn't show any useful information though, so I can't tell you exactly what's going on.

You probably didn't update your Streamlink config file after upgrading Streamlink to 3.x on Windows:
https://github.com/streamlink/streamlink-twitch-gui/issues/842

**Comment by GoToLoop** [comment]
> You probably didn't update your Streamlink config file after upgrading Streamlink to 3.x on Windows:
> #842

You're right! Actually I had already commented out "rtmp-rtmpdump=rtmpdump.exe".

But it seems like we have to comment out "rtmpdump=C:\Program Files (x86)\Streamlink\rtmpdump\rtmpdump.exe" as well.

After that it started to work again!

However in the attempt to fix the issue I ended up uninstalling & removing all Streamlink Twitch Gui config, and lost the dark theme. =(

The dark theme was still active after upgrading to v2.0.0.

Doesn't Streamlink Twitch Gui have the option to choose a theme anymore?

Anyways, thx a lot for the cue! ^_^
